content/docs/1.6.0/references/helm-values.md

```diff
@@ -5,175 +5,173 @@ weight: 5
-| image.csi.resizer.repository | string | `"longhornio/csi-resizer"` | Specify CSI driver resizer image repository. Leave blank to autodetect |
-| image.csi.resizer.tag | string | `"v1.9.2"` | Specify CSI driver resizer image tag. Leave blank to autodetect |
-| image.csi.snapshotter.repository | string | `"longhornio/csi-snapshotter"` | Specify CSI driver snapshotter image repository. Leave blank to autodetect |
-| image.csi.snapshotter.tag | string | `"v6.3.2"` | Specify CSI driver snapshotter image tag. Leave blank to autodetect. |
-| image.longhorn.backingImageManager.repository | string | `"longhornio/backing-image-manager"` | Specify Longhorn backing image manager image repository |
-| image.longhorn.backingImageManager.tag | string | `"master-head"` | Specify Longhorn backing image manager image tag |
-| image.longhorn.engine.repository | string | `"longhornio/longhorn-engine"` | Specify Longhorn engine image repository |
-| image.longhorn.engine.tag | string | `"master-head"` | Specify Longhorn engine image tag |
-| image.longhorn.instanceManager.repository | string | `"longhornio/longhorn-instance-manager"` | Specify Longhorn instance manager image repository |
-| image.longhorn.instanceManager.tag | string | `"master-head"` | Specify Longhorn instance manager image tag |
-| image.longhorn.manager.repository | string | `"longhornio/longhorn-manager"` | Specify Longhorn manager image repository |
-| image.longhorn.manager.tag | string | `"master-head"` | Specify Longhorn manager image tag |
-| image.longhorn.shareManager.repository | string | `"longhornio/longhorn-share-manager"` | Specify Longhorn share manager image repository |
-| image.longhorn.shareManager.tag | string | `"master-head"` | Specify Longhorn share manager image tag |
-| image.longhorn.supportBundleKit.repository | string | `"longhornio/support-bundle-kit"` | Specify Longhorn support bundle manager image repository |
-| image.longhorn.supportBundleKit.tag | string | `"v0.0.32"` | Specify Longhorn support bundle manager image tag |
-| image.longhorn.ui.repository | string | `"longhornio/longhorn-ui"` | Specify Longhorn ui image repository |
-| image.longhorn.ui.tag | string | `"master-head"` | Specify Longhorn ui image tag |
-| image.openshift.oauthProxy.repository | string | `"quay.io/openshift/origin-oauth-proxy"` | For openshift user. Specify oauth proxy image repository |
-| image.openshift.oauthProxy.tag | float | `4.14` | For openshift user. Specify oauth proxy image tag. Note: Use your OCP/OKD 4.X Version, Current Stable is 4.14 |
-| image.pullPolicy | string | `"IfNotPresent"` | Image pull policy which applies to all user deployed Longhorn Components. e.g, Longhorn manager, Longhorn driver, Longhorn UI |
+| image.csi.attacher.repository | string | `"longhornio/csi-attacher"` | Repository for CSI attacher images. When unspecified, Longhorn uses the default value. |
```

Comment:
`Repository for the CSI attacher image` For every image, they should be singular. Please correct other settings accordingly.